src/routes/auth/providers/utils.ts

```diff
@@ -141,7 +141,7 @@ export const initProvider = <T extends Strategy>(
       id,
       email: emails?.[0].value,
       display_name: displayName,
-      avatar_url: photos?.[0].value
+      avatar_url: photos?.[0].value || emails?.[0] && getGravatarUrl(emails[0].value)
```

Comment:
Please improve readability 🙏

Reply:
You could pass `emails?.[0].value` directly as the argument and return `null` if the variable is not found.